packages/fast-components-react-base/src/checkbox/examples.data.ts

```diff
@@ -0,0 +1,38 @@
+import { ICategoryItemProps } from "@microsoft/fast-development-site-react";
+import { ISnapshotTestSuite } from "@microsoft/fast-jest-snapshots-react";
+import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+import Checkbox, { CheckboxHTMLTags, ICheckboxHandledProps,  ICheckboxManagedClasses, ICheckboxUnhandledProps  } from "./checkbox";
+import * as React from "react";
+
+const classes: ICheckboxManagedClasses = {
+    managedClasses: {
+        checkbox: "checkbox",
+        checkbox_input: "checkbox-input",
+        checkbox_label: "checkbox-label",
+        checkbox_disabled: "checkbox-disabled"
+    }
+};
+
+const examples: ISnapshotTestSuite<ICheckboxHandledProps & ICheckboxManagedClasses> = {
+    name: "checkbox",
+    component: Checkbox,
+    data: [
+        {
+            ...classes
+        },
+        {
+            ...classes,
+            tag: "foobar" as any
```

Comment:
do we need to keep this?

Reply:
Yes - think of these as test fixtures. This fixture ensures that when an invalid input is passed the component corrects for that invalid data. The intent of each example will become more clear when we adjust the structure of these to include a description - we have a task for that, we just need to do it.